Add input_weekday integration (#41248)
home-assistant/home-assistant.github.io: source/_integrations/input_weekday.markdown

---
title: Input weekday
description: Instructions on how to use the input weekday helper with Home Assistant.
ha_category:
  - Automation
  - Helper
ha_release: 2025.11
ha_quality_scale: internal
ha_codeowners:
  - '@home-assistant/core'
ha_domain: input_weekday
ha_integration_type: helper
---

> The **Input weekday** helper integration allows you to define a selection of weekdays that can be controlled via the user interface and used within conditions and triggers of an {% term automation %}. This makes it easy to dynamically manage weekday-based automations without having to manually specify weekdays in your automation configuration.

Reviewer: ```suggestion
The **Input weekday** helper {% term integration %} allows you to define a selection of weekdays that can be controlled via the user interface. You can then use conditions and triggers on these days for {% term automation %}. This makes it easy to dynamically manage weekday-based automations without having to manually specify weekdays in your automation configuration.
```
tiny rephrase. Fell free to tweak if you don't like it.

## Configuration

> The preferred way to configure input weekday helpers is via the user interface,
> in which they are known as Weekday Helpers. To add one, go to
> **{% my helpers title="Settings > Devices & services > Helpers" %}** and click the add button;
> next choose the **{% my config_flow_start domain="input_weekday" title="Weekday" %}** option.

Reviewer: ```suggestion
To add a **Weekday Helper**, go to
**{% my helpers title="Settings > Devices & services > Helpers" %}** and select **Add**.
Next, choose the **{% my config_flow_start domain="input_weekday" title="Weekday" %}** option.
```

> To be able to add **Helpers** via the user interface you should have

Reviewer: Do you think we can expect the majority of people to have that in their config? If so, I would move this section into a troubleshooting section further down the page.
